lib/exchange/sideshift/sideshift_exchange_provider.dart

```diff
@@ -210,6 +248,18 @@ class SideShiftExchangeProvider extends ExchangeProvider {
     final min = double.tryParse(responseJSON['min'] as String? ?? '');
     final max = double.tryParse(responseJSON['max'] as String? ?? '');
 
+    if (isFixedRateMode) {
+      final currentRate = await fetchRate(
+        from: to,
+        to: from,
+        isFixedRateMode: isFixedRateMode,
+      );
+      return Limits(
+        min: min != null ? (min * currentRate) : null,
+        max: max != null ? (max * currentRate) : null,
+      );
+    }
+
```

Comment:
1. `fetchRate` function is calling the same API, so why not just use the response we have directly?
2. why are we changing the min/max values if it's a fixed rate mode?